crates/pessimistic-proof-test-suite/src/forest.rs

```diff
@@ -86,7 +86,7 @@ impl Forest {
 
         // Append all the leafs in LET A (mainnet)
         for exit in &exits {
-            self.local_exit_tree_data_a.add_leaf(exit.hash());
+            self.local_exit_tree_data_a.add_leaf(exit.hash()).unwrap();
```

Comment:
This is a naked `unwrap` that creates a potential panic. The thing about error handling in rust is that you have to use it everywhere. If this is a high enough level in the stack to properly handle a failed result, then add a `match` or `or_else` to execute some code to handle/log the error. If this isn't high enough in the stack then you need to change the `imported_bridge_exits` function to return a Result and change this to `self.local_exit_tree_data_a.add_leaf(exit.hash())?;`

Reply:
This function (and everything else in the `pessimistic-proof-test-suite` crate) is only used for testing, so I think it's fine to use `unwrap` here.

Reply:
@wborgeaud I'm ok with the `unwrap` in tests suite, but could you replace it with an `expect` and a more relevant message in case it happens?